### PR TITLE
URDF Cleanup

### PR DESCRIPTION
**Pull request by christiandebuys**
Removes many unused urdfs. Some of them are saved on a separate branch called urdf-backup which will never be merged.

### CODE REVIEW

File: barrett_description/urdf/BarrettHands_WithDumbbells.urdf [hunk 1]
```diff
@@ -0,0 +1,429 @@
+<?xml version = "1.0" ?>
+<robot name="Alex">
+
+	<!-- Gripper Fixed Joints -->
+	<joint name="LeftGrippertAttachmentFixedJoint" type="fixed">
```
Comment (bshrewsbury) [inline barrett_description/urdf/BarrettHands_WithDumbbells.urdf:5]: Can we move the extra weight into the barrett hand urdf as a fixed joint and link, but commented out and delete this extra URDF?

File: alex_description/urdf_backup/20240808_Alex_FixedSpine_SakeHands.urdf
Comment (bshrewsbury) [inline alex_description/urdf_backup/20240808_Alex_FixedSpine_SakeHands.urdf:1]: Can we remove this? Fixed spine nubs is enough

**Review by bshrewsbury — APPROVED** [review]

File: ability_hand_description/urdf/PsyonicHands.urdf [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<?xml version = "1.0" ?>
+<robot name="Alex">
```
Comment (achintyamohan) [inline ability_hand_description/urdf/PsyonicHands.urdf:2]: Should the robot names for the hand urdfs still be Alex? I feel like we should name them PsyonicHand or something along those lines

File: barrett_description/urdf/BarrettHands_WithDumbbells.urdf [hunk 3]
```diff
@@ -0,0 +1,429 @@
+<?xml version = "1.0" ?>
+<robot name="Alex">
```
Comment (achintyamohan) [inline barrett_description/urdf/BarrettHands_WithDumbbells.urdf:2]: Same as Psyonic, should the root element still be Alex?

File: flat_hand_description/urdf/FlatHands.urdf [hunk 4]
```diff
@@ -0,0 +1,49 @@
+<?xml version = "1.0" ?>
+<robot name="Alex">
```
Comment (achintyamohan) [inline flat_hand_description/urdf/FlatHands.urdf:2]: Same as the other two, should this still be named Alex?

**Review by achintyamohan — CHANGES_REQUESTED** [review]

File: ability_hand_description/urdf/PsyonicHands.urdf (hunk 2 above)
Comment (bshrewsbury) [reply ability_hand_description/urdf/PsyonicHands.urdf:2]: I think it should still be alex, I think this is the name that is used in yovariables, ros topics, all kinds of unexpected places

**Review by bshrewsbury — APPROVED** [review]

**Review by dhruvthanki — APPROVED** [review]

File: ability_hand_description/urdf/PsyonicHands.urdf (hunk 2 above)
Comment (christiandebuys) [reply ability_hand_description/urdf/PsyonicHands.urdf:2]: These urdfs are just to keep track of the hands and how they are intended to mount (gripper fixed joints). If we delete all urdfs with a certain hand, someone could still refer to these to see how the hands can be included in a urdf.

Comment (achintyamohan) [reply ability_hand_description/urdf/PsyonicHands.urdf:2]: okay, I doubt it matters that much.

**Review by achintyamohan — APPROVED** [review]

**Pull request merged**